Library/Formula/clang-format.rb

```diff
@@ -52,7 +58,16 @@ def install
       system "ninja", "clang-format"
       bin.install "bin/clang-format"
     end
-    bin.install "tools/clang/tools/clang-format/git-clang-format"
+    path = "tools/clang/tools/clang-format"
+    bin.install "#{path}/git-clang-format"
+    dest = share/"clang"
+    if build.with? "bbedit"
+      dest.install "#{path}/clang-format-bbedit.applescript"
+    end
+    dest.install "#{path}/clang-format-diff.py" if build.with? "diff"
+    dest.install "#{path}/clang-format-sublime.py" if build.with? "sublime"
+    dest.install "#{path}/clang-format.el" if build.with? "emacs"
+    dest.install "#{path}/clang-format.py" if build.with? "vim"
```

Comment:
Unfortunately there is, see: https://llvm.org/svn/llvm-project/cfe/trunk/tools/clang-format/.
It is possible to glob with tools/clang/tools/clang-format/*clang-format*